src/Kobo/System.hpp

```diff
@@ -71,3 +71,12 @@ KoboGetBacklightBrightness();
 
 void
 KoboSetBacklightBrightness(int percent);
+
+bool
+KoboCanChangeBacklightColour();
+
+int
+KoboGetBacklightColour();
```

Comment:
What does the return value mean? Why is it signed? Can it throw exceptions?

Reply:
The return value is an integer from 0 to 10 where 0 is lowest colour temperature (yellowish) and 10 is highest (white/blueish).  The actual number is not any known scale (like degrees K or percent).

The return value is only signed because it gets its value from atoi() which returns int.  Do you have a preferred designation (uint8_t?).  If you wish me to change it, would you like me to change the Brightness code (similar but 0-100)  to your preferred type as well?

I don't see how it could throw an exception.  It reads a string from the driver (which is noexcept) and calls atoi() which returns 0 in the case of invalid.

Reply:
The explanation belongs in a doxygen-style code comment.

Reply:
Will gladly do this.  Can you point me to a preferred doxygen sample as I would like to conform to this repo's standard?  Do you want the entire function def documented or just the colour return value (get) / call parameter (set)?

Also, would you like me to document both functions (set and get) as they both use colour?

Reply:
I went ahead and did the code comment in the same style as some other (src/Engine/GlideSolvers/GlidePolar.cpp).  I commented both KoboGetBacklightColour and KoboSetBacklightColour and KoboGetBacklightColourFile.  I also added noexcept to all the ...BacklightColour... functions as none of them can cause an exception other than OOM.